openapi/endpoints/accounts/accounts-create.yaml

```diff
@@ -19,20 +19,17 @@ responses:
           $ref: './models/create-account-response.yaml'
   '403':
     description: |
-      User is not allowed to perform the action with the reason stated in the `message`
+      User is not allowed to perform the action with the reason stated in the `errorCode`
 
       **FORBIDDEN**
       Insufficient permissions to perform the action.
 
       **INVALID_PARTNER_ACCOUNT**
-      The partner account does not exist.
+      The account is not of type partner.
 
       **INVALID_PARENT_ACCOUNT**
-      The parent account does not exist or is not a partner account.
-
-      **INVALID_USER**
-      The user does not exist or is not fully provisioned.
+      The account is not of type partner.
```

Comment:
💬 Description for both codes (INVALID_PARTNER_ACCOUNT and INVALID_PARENT_ACCOUNT) is identical now. Might create confusion why do we need two different error codes with the same reason.

Reply:
Agreed with Roman B to keep codes distinct but update the description here in the docs.